packages/joint-core/src/dia/Graph.mjs

```diff
@@ -429,35 +423,83 @@ export const Graph = Model.extend({
         this.stopBatch(batchName);
     },
 
+    addCellLayer(cellLayer, opt) {
+        this.cellLayersController.addCellLayer(cellLayer, opt);
+    },
+
+    insertCellLayer(cellLayer, insertBefore) {
+        this.cellLayersController.insertCellLayer(cellLayer, insertBefore);
+    },
+
+    removeCellLayer(cellLayer, opt) {
+        this.cellLayersController.removeCellLayer(cellLayer.id, opt);
+    },
+
+    getDefaultCellLayer() {
+        return this.cellLayersController.getDefaultCellLayer();
+    },
+
+    setDefaultCellLayer(layerId) {
+        this.cellLayersController.setDefaultCellLayer(layerId);
+    },
+
+    getCellLayer(layerId) {
+        return this.cellLayersController.getCellLayer(layerId);
+    },
+
+    hasCellLayer(layerId) {
+        return this.cellLayersController.hasCellLayer(layerId);
+    },
+
+    getCellLayers() {
+        return this.cellLayersController.getCellLayers();
+    },
+
+    getCellLayerCells(layerId) {
+        return this.cellLayersController.getCellLayerCells(layerId);
+    },
+
     // Get a cell by `id`.
     getCell: function(id) {
 
-        return this.get('cells').get(id);
+        return this.cellCollection.get(id);
     },
 
     getCells: function() {
-
-        return this.get('cells').toArray();
+        // Preserve old order without layers
```

Comment:
What does `old order` mean? The reader would not know.

Reply:
Did comment it properly